### ISSUE TITLE
SC2216: piping to 'du' is legit

### ISSUE THREAD

**Issue by cubanpit** [koalaman/shellcheck#1286]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2216
- My shellcheck version (`shellcheck --version` or "online"): version 0.5.0 && online

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
find . -print0 | du --files0-from=-

```

#### Here's what shellcheck currently says:
```SC2216: Piping to 'du', a command that doesn't read stdin. Wrong command or missing xargs?```

#### Here's what I wanted or expected to see:
No warning, because this is legit with `du`, from manual page:
```
--files0-from=F
              summarize disk usage of the NUL-terminated file names specified in file F;
              if F is -, then read names from standard input
```

**Comment by szepeviktor** [comment]
@koalaman Thanks.
In which version will it be released?

**Comment by koalaman** [comment]
This fix has not been merged, see https://github.com/koalaman/shellcheck/pull/1357

**Comment by szepeviktor** [comment]
Thank you.
